packages/binding-http/src/http-server.ts

```diff
@@ -124,10 +121,12 @@ export default class HttpServer implements ProtocolServer {
                     this.httpSecurityScheme = "Bearer";
                     break;
                 case "oauth2":
-                    this.httpSecurityScheme = "OAuth";
-                    const oAuthConfig = config.security as OAuth2ServerConfig;
-                    this.validOAuthClients = new RegExp(oAuthConfig.allowedClients ?? ".*");
-                    this.oAuthValidator = createValidator(oAuthConfig.method);
+                    {
+                        this.httpSecurityScheme = "OAuth";
+                        const oAuthConfig = config.security as OAuth2ServerConfig;
+                        this.validOAuthClients = new RegExp(oAuthConfig.allowedClients ?? ".*");
+                        this.oAuthValidator = createValidator(oAuthConfig.method);
+                    }
```

Comment:
Unnecessary change. At least I can not spot the reason

Reply:
This was to fix a ESLint error:
![image](https://user-images.githubusercontent.com/680220/136521189-c30a9e94-be3c-4508-9e9c-9d04a2bd6635.png)